CMakeLists.txt

```diff
@@ -124,21 +124,25 @@ if(VN_SDK_DOCS)
     lcevc_add_subdirectory(docs/sphinx)
 endif()
 
+include(GNUInstallDirs)
+
 # Record configuration - header and README
```

Comment:
Update comment to mention/explain the new configuration file?

Reply:
It's not really a configuratioin file, just a set of defaults based on the prefix?

Reply:
Fair. Possibly a better comment might be:

`# Generate the config header, README, and pkgconfig defaults`